quickstarts/biztalk360/config.yml

```diff
@@ -0,0 +1,80 @@
+id: 4bd2976c-2180-4b22-8539-a00038a7738c
+# Name of the quickstart (required)
+name: biztalk360
+
+# Displayed in the UI (required)
+title: BizTalk360 
+
+# Long-form description of the quickstart (required)
+description: |
+  ## What's BizTalk360?  
+  BizTalk360 is a one-stop tool for administration, monitoring and application performance management (APM) of BizTalk environments.
+
+  To be able to use the integration with New Relic from BizTalk360, you must have :
+  - New Relic License to be configured in BizTalk360 Application. Download the latest version of BizTalk360 from [here](http://www.biztalk360.com/free-trial/)
+  - BizTalk360 Platinum license .
+  - You must be a Super User in your BizTalk360 application to configure the New Relic environment in BizTalk360
+
+  Below are some of the important performance metrics you can see in your Newrelic dashboard.  
+    
+  - BizTalk and SQL Server Health
+    - CPU Usage
+    - Memory Usage
+    - Disk Free Space
+    - Average Disk Queue Length
+    - Network Performance
+
+  - Host Performance
+    - Host Instance performance by CPU
+    - Host Instance Performance by Memory
+    - CPU Consuming Host Instances
+    - Top 10 Memory Consuming Host Instances
+
+  - BizTalk Messaging Performance
+    - BizTalk Host Performance
+    - Documents Receive/Second
+    - Documents Processed/Second
+    - Inbound Latency (Sec)
+    - Outbound Latency (Sec)
+    - Outbound Adapter Latency (Sec)
+
+  - Throttling Performance
+    - Message delivery Throttling State
+    - Message Publishing Throttling State
+    - Message Delivery Outgoing Rate
+    - Message Delivery Incoming Rate
+    - Active Instance Count  
+
+
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Integrate BizTalk Performance data with NewRelic from BizTalk360.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - BizTalk360
+  - Raja Sreenivasan
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - biztalk360
+  - integration
+  - biztalk server  
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - infra-agent-standard
```

Comment:
Forgot to remove the install Plans. We use the Metric API to send the metrics from the BizTalk360 application. So once the metrics are started to send from the BizTalk360 application users can use this QuickStart to build the dashboard

Reply:
@raja2raj02 Ok thanks for confirming. As an FYI: without an installplan defined the quickstart will just link to the documentation URL defined when the user clicks on `see installation docs`, it will no invoke the guided install flow.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#install-plans

If that is what you expect to occur on this quickstart just let me know and I can merge this in.

Reply:
Yes @jpvajda, as you mentioned I am expecting the same behaviour. You can merge this in.